diff -uNr linux-source-4.9-orig/drivers/gpu/drm/gma500/backlight.c linux-source-4.9/drivers/gpu/drm/gma500/backlight.c
--- linux-source-4.9-orig/drivers/gpu/drm/gma500/backlight.c	2017-11-24 07:33:43.000000000 +0000
+++ linux-source-4.9/drivers/gpu/drm/gma500/backlight.c	2018-01-13 10:30:17.343273229 +0000
@@ -26,6 +26,8 @@
 #include "intel_bios.h"
 #include "power.h"
 
+#undef CONFIG_BACKLIGHT_CLASS_DEVICE  // OpenFrame Workaround
+
 #ifdef CONFIG_BACKLIGHT_CLASS_DEVICE
 static void do_gma_backlight_set(struct drm_device *dev)
 {
diff -uNr linux-source-4.9-orig/drivers/gpu/drm/gma500/psb_device.c linux-source-4.9/drivers/gpu/drm/gma500/psb_device.c
--- linux-source-4.9-orig/drivers/gpu/drm/gma500/psb_device.c	2017-11-24 07:33:43.000000000 +0000
+++ linux-source-4.9/drivers/gpu/drm/gma500/psb_device.c	2018-01-13 10:30:17.343273229 +0000
@@ -75,6 +75,8 @@
 	uint32_t value;
 	uint32_t blc_pwm_precision_factor;
 
+	return 0;  // OpenFrame Workaround
+
 	/* get bl_max_freq and pol from dev_priv*/
 	if (!dev_priv->lvds_bl) {
 		dev_err(dev->dev, "Has no valid LVDS backlight info\n");
